slidechain/schema.go

```diff
@@ -30,14 +30,8 @@ CREATE TABLE IF NOT EXISTS pegs (
 
 CREATE TABLE IF NOT EXISTS exports (
   txid BLOB NOT NULL PRIMARY KEY,
-  exporter TEXT NOT NULL,
-  amount INTEGER NOT NULL,
-  asset_xdr BLOB NOT NULL,
-  temp_addr TEXT NOT NULL,
-  seqnum INTEGER NOT NULL,
   pegged_out INTEGER NOT NULL DEFAULT 0,
-  anchor BLOB NOT NULL,
-  pubkey BLOB NOT NULL
+  ref BLOB NOT NULL
```

Comment:
This is json-encoded, right? So it can be `TEXT` (though not sure if that matters), and it should have a name that indicates its purpose and encoding, like `pegout_json` or something.